modules/ROOT/pages/create-community.adoc

```diff
@@ -219,6 +219,10 @@ These are the permissions for the member users:
 |*Read*
 |*Read Access*
 
+|CommunityAssetVersions
```

Comment:
@wwarner-salesforce here we could add some * or a clarification to notify the user that some fields were introduced in the last version of the docs and point them to the release notes or the upgrade page.
- if you are in an older version you wont' see all objects. in your org"

Reply:
Updated: "These permissions reflect the current API Community Manager package version. If you are using an older version, your member user profile may have only some of these permissions objects. Set the objects in your member user profile to match the table, and ignore objects that are not in your member user profile."